src/index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+import _, { intersection } from 'lodash'; /* eslint-enable */
+import './style.css';
+import Interactions from './modules/interactions.js';
+import Tasks from './modules/tasks.js';
+
+document.querySelector('#formTask').addEventListener('click', () => {
+  Interactions.addTask();
+  const input = document.querySelector("#formTask input[type='text']");
+  input.value = '';
+});
+
+document.querySelector('#formTask').addEventListener('submit', () => {
+  Interactions.addTask();
+  const input = document.querySelector("#formTask input[type='text']");
+  input.value = '';
+});
+
+const checkLocalInput = () => {
+  const data = JSON.parse(localStorage.getItem('data'));
+  if (data !== null) {
+    Tasks.initializeTasks(data);
+    for (let i = 0; i < data.length; i += 1) {
+      if (Interactions.hasValue(data[i])) {
+        Interactions.updateDisplay(data[i]);
+      }
+    }
```

Comment:
I think the code can be improved by using a forEach() statement.

Reply:
Thanks! done ✔